cpp/extract_normdata_translations.cc

```diff
@@ -0,0 +1,225 @@
+
+/* We would like to have a tool that extracts translations for given
```

Comment:
Copyright is missing.  Also please state what the tool does not what we would like to have.